src/test/features/dashboard/routes/claimStatus/part-admission.ts

```diff
@@ -706,13 +707,8 @@ const legacyClaimDetails = [
       claimantRespondedAt: MomentFactory.currentDate(),
       ...directionsQuestionnaireDeadline
     },
-    claimantAssertions: ['You’ve rejected John Doe’s response and said you want to take the case to court.',
-      'Tell us your hearing requirements',
-      'complete a directions questionnaire form',
-      'You need to',
-      'to tell us more about the claim.',
-      'Your claim won’t proceed if you don’t complete and return the form before',
-      'Tell us you’ve settled'
+    claimantAssertions: ['Wait for the court to review the case',
+      'You’ve rejected ' + partAdmissionClaim.claim.defendants[0].name + '’s response and said you want to take the case to court.'
     ],
```

Comment:
You've added 6 new statuses above but I'm not seeing any new route tests for them?